test/functional/apps/visualize/index.ts

```diff
@@ -67,20 +70,22 @@ export default function({ getService, loadTestFile }: FtrProviderContext) {
 
       loadTestFile(require.resolve('./_line_chart'));
       loadTestFile(require.resolve('./_pie_chart'));
-      loadTestFile(require.resolve('./_region_map'));
       loadTestFile(require.resolve('./_point_series_options'));
       loadTestFile(require.resolve('./_markdown_vis'));
       loadTestFile(require.resolve('./_shared_item'));
       loadTestFile(require.resolve('./_lab_mode'));
       loadTestFile(require.resolve('./_linked_saved_searches'));
       loadTestFile(require.resolve('./_visualize_listing'));
+      if (!isOss) {
```

Comment:
**question**: why does this PR need to skip these tests for OSS?

Reply:
I think it's that OSS still has coordinate map (previously TileMap) and region map, but in default distribution they are replaced by the Maps app

Reply:
That makes sense. Shouldn't `master` be failing though, if these tests are presumably running there? Or was there a different safeguard that prevented these tests from running?

Reply:
Jenkins only currently OSS tests against OSS Kibana.  But this PR supports running OSS tests (like these visualizations) against default distribution.  This is also like Cloud.